agent:
  planner:
    time_horizon: 10.0 # seconds
    y_resolution: 0.3
    y_horizon: 10.0 #m
    vel_resolution: 0.2
  computation_time:
    mean: 10
    std_dev: 1.5
